CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * Added more standard fields such as `description` to the emulator package
 * Added `import-map-webpack-plugin` to the `piral-cli-webpack`
 * Added more flexibility to the `piral declaration` command (#316)
+* Added ability to set and retrieve state when signing in using `piral-oidc`
```

Comment:
Could we move this into the 1.0.0 section? 0.12.0 is already released. I'll then move the (so far) 1.0.0 items to 0.12.1.

Reply:
Oops, my bad. Sorted!